packages/textlint/test/textlint-module-resolver/textlint-module-resolver-test.js

```diff
@@ -81,9 +81,9 @@ describe("textlint-module-resolver", function() {
     describe("#resolvePluginPackageName", function() {
         it("should resolve plugin package name", function() {
```

Comment:
Originally, this test case aim to test non scoped plugin pacakge name.

Current test description should be renamed.

```diff
- should resolve plugin package name"
+ should resolve scoped plugin package name"
```

Reply:
Can you add a test case that is non scoped test case using `textlint-plugin-html`.

```js
        it("should resolve plugin package name", function() {
            const resolver = createResolve();
            const shortPkg = resolver.resolvePluginPackageName("html");
            assert.equal(typeof shortPkg, "string");
            const longPkg = resolver.resolvePluginPackageName("textlint-plugin-html");
             assert.equal(typeof longPkg, "string");
             assert.equal(shortPkg, longPkg);
         });
```

Reply:
@azu, thank you! Modify the description and add test case according to your review.